src/chocolatey/infrastructure.app/validations/GlobalConfigurationValidation.cs

```diff
@@ -64,7 +64,7 @@ private void ValidateUseOfPackageExitCode(ChocolateyConfiguration config, IColle
                     Message = @"When attempting to halt execution of a Chocolatey command based on a
    request for a system reboot, it is necessary to have the
    usePackageExitCodes feature enabled.  Use the following command:
-     choco feature enable -name={0}
+     choco feature enable --name={0}

```

Comment:
Should we ensure that the name is quoted as well?

IE:

```suggestion
     choco feature enable --name=""{0}""
```

Reply:
I did think about this, but I also thought no.  Feature names, or config values, never have any spaces in them, so the need for quoting goes away in my opinion.

If we were to do this though, it would be `choco feature enable --name="'{0}'"`